controllers/cluster_controller.go

```diff
@@ -608,6 +613,10 @@ func getStartCommandLines(cluster *fdbtypes.FoundationDBCluster, processClass st
 		confLines = append(confLines, fmt.Sprintf("tls_verify_peers = %s", cluster.Spec.MainContainer.PeerVerificationRules))
 	}
 
+	if cluster.NeedsExplicitListenAddress() {
+		confLines = append(confLines, fmt.Sprintf("listen_address = %s", cluster.GetFullAddressList("$FDB_POD_IP", false)))
```

Comment:
Is there any reason we don't want to add the flag by default?

Reply:
We don't want to make any introduce any changes to a cluster by default with new versions of the operator, as it would cause uncontrolled changes across all the clusters. We may want to add it across the board in the future to avoid needless variation between different configurations. I think we can tackle that in its own issue, though.